entrypoint.sh

```diff
@@ -107,27 +93,17 @@ restore() {
 list-backups() {
     local batch_mode=$([[ "$1" = "--batch-mode" ]] && echo true || echo false)
     local files_in_s3=();
-    if is_aws_s3_enabled; then
+    if s3sync is_enabled; then
```

Comment:
This and the other .. I have rejigged a bit so that the script will fail before it does anything.
At that point, it will also print out an `info` line if the functionality is not configured .. i.e all variables are empty.